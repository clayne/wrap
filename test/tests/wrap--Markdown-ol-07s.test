wrap | /dev/null | -u | md-ol-07s.md | 0
