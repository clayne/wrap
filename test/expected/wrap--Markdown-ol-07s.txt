1234567890. This is not a list item.  1234567891. And neither is this.
